trade-finance-manager-ui/server/controllers/utilisation-reports/record-corrections/initiate-record-correction-request/index.ts

```diff
@@ -48,14 +48,14 @@ export const validateRecordCorrectionRequestFeeSelections = (checkedCheckboxIds:
   const checkedCheckboxId = checkedCheckboxIds[0];
   const selectedCheckboxStatus = getFeeRecordStatusFromPremiumPaymentsCheckboxId(checkedCheckboxId);
 
-  if (selectedCheckboxStatus !== FEE_RECORD_STATUS.TO_DO) {
+  if (!(selectedCheckboxStatus === FEE_RECORD_STATUS.TO_DO || selectedCheckboxStatus === FEE_RECORD_STATUS.TO_DO_AMENDED)) {
```

Comment:
that previously talked about helper could live in libs/common and be used here too